shunting-yard-common/src/main/java/com/hotels/shunting/yard/common/emitter/AbstractMetaStoreEventListener.java

```diff
@@ -81,44 +83,72 @@ private Message withPayload(SerializableListenerEvent event) throws MetaExceptio
 
   @Override
   public void onCreateTable(CreateTableEvent tableEvent) throws MetaException {
-    LOG.info("Create table event received");
-    executorService.submit(message(withPayload(serializableListenerEventFactory.create(tableEvent))));
+    log.info("Create table event received");
+    try {
+      executorService.submit(message(withPayload(serializableListenerEventFactory.create(tableEvent))));
+    } catch (Exception e) {
+      error(e);
```

Comment:
Test that throwing an exception -> is caught and "errored"

Reply:
done